### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by truhin-andrei**

### CODE REVIEW

File: src/pages/director.jsx [hunk 1]
```diff
@@ -29,33 +29,13 @@ const DirectorPage = () => {
...
+  const query = getQueryDataDirectors()
   useEffect(() => {
-    setDirectorData(query.allDirectorsJson.nodes[0].directors[idOfDirector - 1])
+    setDirectorData(query.directors[idOfDirector - 1])
   })
-
   const handleDirectorChange = (next = true) => {
     setLoadingIMG(true)
-    const length = query.allDirectorsJson.nodes[0].directors.length
+    const length = query.directors.length
```
Comment (rss-maksim) [inline src/pages/director.jsx]: You can use destruction
```
 const { length } = query.directors
```

File: src/pages/index.js [hunk 2]
```diff
@@ -1,21 +1,93 @@
...
+/* eslint-disable array-bracket-spacing */
+import React, { useState, useEffect } from 'react'
+import { Paper } from '@material-ui/core'
+import Grid from '@material-ui/core/Grid'
+import { useMediaQuery } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import CircularProgress from '@material-ui/core/CircularProgress'
+import DirectorCard from '../components/DirectorCard/DirectorCard'
+import getQueryDataDirectors from '../directors/getQueryDataDirectors'
+import { useTranslation } from 'react-i18next'
```
Comment (rss-maksim) [inline src/pages/index.js]: It's better to separate local and npm packages

File: src/pages/index.js [hunk 3]
```diff
@@ -1,21 +1,93 @@
...
+}))
+const IndexPage = () => {
+  const [loadingIMG, setLoadingIMG] = useState(true)
+  const [directorData, setDirectorData] = useState(null)
+  const { t } = useTranslation()
+  const match = useMediaQuery('(max-width: 945px)')
+  const classes = useStyles()
+  const query = getQueryDataDirectors()
+  useEffect(() => {
+    setDirectorData(query.directors[2 - 1])
```
Comment (rss-maksim) [inline src/pages/index.js]: Don't understand ```[2 - 1]```

**Review by rss-maksim — APPROVED** [review]

File: src/pages/director.jsx (hunk 1 above)
Comment (truhin-andrei) [reply src/pages/director.jsx]: done

File: src/pages/index.js (hunk 2 above)
Comment (truhin-andrei) [reply src/pages/index.js]: done

File: src/pages/index.js (hunk 3 above)
Comment (truhin-andrei) [reply src/pages/index.js]: It should be random index for a day. Change for just random index

**Pull request merged**